src/bin/pgcopydb/schema.c

```diff
@@ -971,14 +971,19 @@ struct FilteringQueries listSourceTablesSQL[] = {
 		"               ("
 		"                  select attnum, atttypid::integer, "
 		"                         format('%I', attname) as attname, "
-		"                         i.indrelid is not null as attisprimary "
+		"                         i.indrelid is not null as attisprimary, "
+		"						  col.is_generated = 'ALWAYS' as attisgenerated "
 		"                    from pg_attribute a "
 		"                         left join pg_index i "
-		"                                on i.indrelid = a.attrelid "
+		"                               on i.indrelid = a.attrelid "
```

Comment:
Can you cleanup your PR to avoid useless whitespace changes? (the spacing here was not a mistake btw, it was intended that way).